linux/cli.py

```diff
@@ -455,7 +465,7 @@
 ## Start of clicker code
 shouldClick = False # controlls the constantclick
 
-
+## No idea why you get a bug here hehe not my code not my problem @spamix
```

Comment:
What Bug?

Reply:
I dont really know, i just know that i got an error message when running it with Python

Reply:
...And i didnt change anything there so its your fault :)

Reply:
@SpamixOfficial I get error `Key.f2 to click constantly (toggle on/off by clicking the key!)
Esc to exit!
Traceback (most recent call last):
  File "/home/user/Documents/coole sachen/openclick/linux/cli.py", line 469, in <module>
    @listener.event
     ^^^^^^^^
NameError: name 'listener' is not defined. Did you mean: 'Listener'?`

when running `python linux/cli.py `

Reply:
I'll take a look later when I get some time

Reply:
I just checked your last push, everthing works there.
I dont see anything where I changed anything related to that event, thats odd.

Reply:
Weird

Reply:
@SpamixOfficial I just checked again, your last push on dev is causing the error. So its not my fault hehe.


```
python cli.py

  /$$$$$$                                 /$$$$$$  /$$ /$$           /$$
 /$$__  $$                               /$$__  $$| $$|__/          | $$
| $$  \ $$  /$$$$$$   /$$$$$$  /$$$$$$$ | $$  \__/| $$ /$$  /$$$$$$$| $$   /$$
| $$  | $$ /$$__  $$ /$$__  $$| $$__  $$| $$      | $$| $$ /$$_____/| $$  /$$/
| $$  | $$| $$  \ $$| $$$$$$$$| $$  \ $$| $$      | $$| $$| $$      | $$$$$$/
| $$  | $$| $$  | $$| $$_____/| $$  | $$| $$    $$| $$| $$| $$      | $$_  $$
|  $$$$$$/| $$$$$$$/|  $$$$$$$| $$  | $$|  $$$$$$/| $$| $$|  $$$$$$$| $$ \  $$
 \______/ | $$____/  \_______/|__/  |__/ \______/ |__/|__/ \_______/|__/  \__/
          | $$
          | $$
          |__/


							SpamixOfficial 2023
Controls: 
Key.f1 to click (hold to click!) 
Key.f2 to click constantly (toggle on/off by clicking the key!)
Esc to exit!
Traceback (most recent call last):
  File "/tmp/OpenClick/linux/cli.py", line 460, in <module>
    @listener.event
     ^^^^^^^^
NameError: name 'listener' is not defined. Did you mean: 'Listener'?


```

Reply:
Well, would you mind changing it to "Listener" then?